### DISCUSSION TITLE
sharing > allow downloads? no option displaying

### DISCUSSION THREAD

**Discussion by blogbourri** [navidrome/navidrome#2381] [Q&A] [answered]
Hello, new to Navidrome and loving it so far! One feature does not seem to be working as described in the documents. I'm running Navidrome 0.49.3 in Docker on a Synology NAS. I have the following environment variables set:

ND_ENABLESHARING: true
ND_ENABLEDOWNLOADS: true

I see on [the docs](https://www.navidrome.org/docs/usage/sharing/) that it should show an "Allow Downloads?" toggle when I go to share an album, but that option doesn't show up for me. Downloads elsewhere in Navidrome work as expected. Is this a bug, or is it possible I set something up incorrectly? I saw a closed ticket from when the feature was first added where someone else had the same issue but couldn't locate a resolution to it.

**Comment by deluan** [comment, marked as answer]
Sorry, the doc does not reflect the current version 0.49.3. Share downloads will be available in the next release. For now, if you want to give it a try, you can use the latest dev build.

EDIT: Updated the screenshot to reflect the current version and avoid confusion. Thanks for bringing that to my attention.
